rules/eslint/style/walmart.js

```diff
@@ -56,6 +60,8 @@ module.exports = {
     "no-nested-ternary": 2,
     // disallow use of the Object constructor
     "no-new-object": 2,
+    // disallow use of certain syntax in code
+    "no-restricted-syntax": 0,
```

Comment:
This is neat!  Can't think of anything to turn on here, but good to know it is available.